docs/software/SRS.rst

```diff
@@ -580,7 +580,7 @@ Controller's pre-test mode must include the following checks: ...
 GUI
 ^^^
 
-.. item:: RW-SW-2
+.. item:: RW-SW-162
```

Comment:
this is the second RW-SW-162 (duplicate from line 559)

Reply:
Nice catch! Fixed.